innopoints/views/product.py

```diff
@@ -101,8 +105,18 @@ def create_product():
         log.exception(err)
         abort(400, {'message': 'Data integrity violated.'})
 
-    out_schema = ProductSchema(exclude=('notifications',
-                                        'varieties.product_id',
+    # TODO: replace the following with proper debounce
+    # Check if a notification has been sent today
+    query = Notification.query.filter(
+        Notification.type == 'new_arrivals',
```

Comment:
Hardcoded string here? I mean, changing it to the actual reference will not save us from renaming, but it just looks more clean and reassuring, for some reason

Reply:
I agree. I just didn't want to import it just for this. Will change